src/lib/eap_aka_sim/state_machine.c

```diff
@@ -1519,7 +1523,8 @@ static unlang_action_t sim_challenge_request_compose(rlm_rcode_t *p_result, modu
 {
 	eap_aka_sim_session_t	*eap_aka_sim_session = talloc_get_type_abort(eap_session->opaque,
 									     eap_aka_sim_session_t);
-	fr_pair_t		*to_peer = request->reply_pairs, *vp;
+	fr_pair_list_t		to_peer = request->reply_pairs;
```

Comment:
This should be `fr_pair_list_t *`otherwise it's going to break horribly when we switch to a more advanced list head